### PR TITLE
fix double slashes

### PR DESCRIPTION
**Pull request by il-av**
Fix invalid path being used for certificate files

Screenshot_20230317-225315

### CODE REVIEW

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

File: Source/Core/Core/IOS/Network/SSL.cpp [hunk 1]
```diff
@@ -385,6 +385,7 @@ std::optional<IPCReply> NetSSLDevice::IOCtlV(const IOCtlVRequest& request)
     {
       WII_SSL* ssl = &_SSL[sslID];
       const std::string cert_base_path = File::GetUserPath(D_SESSION_WIIROOT_IDX);
+      cert_base_path.pop_back();
       const std::vector<u8> client_cert =
           ReadCertFile(cert_base_path + "/clientca.pem", s_client_cert_hash, m_cert_error_shown);
```
Comment (BhaaLseN) [inline Source/Core/Core/IOS/Network/SSL.cpp:390]: Have you considered removing it from the string instead?
```suggestion
      const std::vector<u8> client_cert =
          ReadCertFile(cert_base_path + "clientca.pem", s_client_cert_hash, m_cert_error_shown);
```

Same for the one below.

(And actually, this should probably use the C++17/20/2z/whatever-equivalent of C#'s `Path.Combine` if I could make a wish; but I don't know if there is anything in `std::filesystem` that does this cheaply)

**Comment by il-av** [comment]
seems like it could handle the double slashes
that warning came from the file not existing and it wasn't being recovered by a system update
regardless it was still using an invalid path.

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Pull request closed without merging**